src/consumer/component-ops/components-diff.js

```diff
@@ -28,8 +29,17 @@ export default (async function componentsDiff(
   try {
     const getResults = (): Promise<DiffResults[]> => {
       if (version && toVersion) {
-        // $FlowFixMe - version, toVersion are string here, the error is unclear
-        return Promise.all(ids.map(id => getComponentDiffBetweenVersions(consumer, tmp, id, version, toVersion)));
+        // we already have the component in the components array, we just need to search for it
+        return Promise.all(
+          ids.map((id) => {
+            if (!id.scope && components) {
+              const foundComponent = lfind(components, o => o.box === id.box && o.name === id.name);
```

Comment:
do we really need to add a new package? Array.find is in the language.
`components.find(o => o.box === id.box && o.name === id.name);` it's even shorter :)